Engine/source/T3D/player.cpp

```diff
@@ -2641,10 +2641,33 @@ void Player::updateMove(const Move* move)
          }
          else
          {
+
             mRot.z += y;
-            // Rotate the head back to the front, center horizontal
-            // as well if we're controlling another object.
-            mHead.z *= 0.5f;
+
+            if(move->noWeaponTPCam) {
```

Comment:
I stopped counting at 40 other occurrences in this file of incorrect braces, hardly fair to pick on him.